remoulade/state/backends/stub.py

```diff
@@ -0,0 +1,30 @@
+import time
+
+from ..backend import StateBackend
+
+
+class StubBackend(StateBackend):
+
+    states = {}
+
+    def get_state(self, message_id):
+        """ Given a message_id returns the value in
+        the database"""
+        message_key = self._build_message_key(message_id)
+        data = self.states.get(message_key)
+        if data:
+            data = self.encoder.decode(data)
+            if time.monotonic() > data["expiration"]:
+                self._delete(message_key)
+                data = None
+            else:
+                data = data["state"]
+        return data
+
+    def set_state(self, message_id, state, ttl):
+        message_key = self._build_message_key(message_id)
+        payload = {"state": state, "expiration": ttl}
```

Comment:
you need to store expiration: time.monotonic() + ttl: here and then in get_state check that `time.monotonic() > data["expiration"] `.

Reply:
cool perfect